CODING_CONVENTIONS_C++.md

```diff
@@ -0,0 +1,369 @@
+You should check out the [C Conventions](CODING_CONVENTIONS.md) as some section
+still apply (Documentation, Git, Travis).
+
+When contributing source code, please adhere to the following coding style,
+which is loosely based on the [Google C++ Style Guide] and the coding
+ conventions used by the C++ Standard Library. Based on the [CAF coding style].
+
+[Google C++ Style Guide]: (http://google-styleguide.googlecode.com/svn/trunk/cppguide.xml)
```

Comment:
The link doesn't work. Should be probably https://google.github.io/styleguide/cppguide.html